src/plugins/intel_cpu/src/mkldnn_exec_network.cpp

```diff
@@ -242,6 +258,78 @@ InferenceEngine::Parameter MKLDNNExecNetwork::GetMetric(const std::string &name)
     }
 }
 
+InferenceEngine::Parameter MKLDNNExecNetwork::GetMetric(const std::string &name) const {
+    if (_graphs.empty())
+        IE_THROW() << "No graph was found";
+    // @todo Can't we just use local copy (_cfg) instead?
+    auto graphLock = GetGraph();
+    const auto& graph = graphLock._graph;
+    const auto& config = graph.getProperty();
+
+    if (isLegacyAPI()) {
+        return GetMetricLegacy(name, graph);
+    }
+
+    auto RO_property = [](const std::string& propertyName) {
+        return ov::PropertyName(propertyName, ov::PropertyMutability::RO);
+    };
+
+    if (name == ov::supported_properties) {
+        return std::vector<ov::PropertyName> {
+            RO_property(ov::model_name.name()),
+            RO_property(ov::optimal_number_of_infer_requests.name()),
+            RO_property(ov::streams::num.name()),
+            RO_property(ov::affinity.name()),
+            RO_property(ov::inference_num_threads.name()),
+            RO_property(ov::enable_profiling.name()),
+            RO_property(ov::hint::inference_precision.name()),
+            RO_property(ov::hint::performance_mode.name()),
+            RO_property(ov::hint::num_requests.name()),
+        };
+    }
+
+    if (name == ov::model_name) {
+        // @todo Does not seem ok to 'dump()' the whole graph everytime in order to get a name
+        return graph.dump()->get_friendly_name();
+    } else if (name == ov::optimal_number_of_infer_requests) {
+        const auto streams = config.streamExecutorConfig._streams;
+        return static_cast<uint32_t>(streams); // ov::optimal_number_of_infer_requests has no negative values
```

Comment:
why not just `return static_cast<uint32_t>(config.streamExecutorConfig._streams)` here and below?

Reply:
Basically to have it in more declarative manner / easier to read
Making it shorter (oneliner) will not make it more readable I think.

Reply:
The better way is to use :
```c++
return decltype(ov::optimal_number_of_infer_requests)::value_type(config.streamExecutorConfig._streams);
```